geode-core/src/integrationTest/java/org/apache/geode/redis/HashesJUnitTest.java

```diff
@@ -172,7 +172,7 @@ private String randString() {
     int length = rand.nextInt(8) + 5;
     StringBuilder rString = new StringBuilder();
     for (int i = 0; i < length; i++)
-      rString.append((char) (rand.nextInt(57) + 65));
+      rString.append((char) (rand.nextInt(25) + 65));
```

Comment:
nextInt('bound') will give a random value from 0 to 'bound' - 1...so this will actually the limit the random strings to [A-Y]